client/src/pages/Login.js

```diff
@@ -54,9 +54,11 @@ class Login extends Component {
         // get jwt token from header
         const token = response.headers["x-auth-token"];
 
-        // add token, username to local storage
+        // add token,user id and username to local storage
         window.localStorage.setItem("token", token);
         window.localStorage.setItem("username", response.data.username);
+        window.localStorage.setItem("id", response.data._id);
```

Comment:
you shouldn't need to put so much information in localstorage - the token actually contains username and id and you should be fetching that data on the front-end, and on the back-end you don't need to send that info to the back-end